### PR TITLE
Chore:hidding the carousel when we select any category

### PR DESCRIPTION
**Pull request by sjyothi54**
JIRA:https://newrelic.atlassian.net/browse/CXPUI-26
Description:When user selects a category in left navigation pane carousel of most popular and featured are supposed to be hidden
**OLD VIEW:**
when All option selected in left panel
image
when any category is selected
image
**NEW VIEW:**
when All option selected in left panel
image
when any category is selected
image

### CODE REVIEW

**Comment by aswanson-nr** [comment]
Quick close/reopen to retrigger that build

File: src/pages/index.jsx [hunk 1]
```diff
@@ -427,10 +432,94 @@ const QuickstartsPage = ({ data, location }) => {
             </Overlay>
           </div>
 
-          {isSearchInputEmpty && (
+          {isSelectCategory && (
```
Comment (josephgregoryii) [inline src/pages/index.jsx:435]: I think we can consolidate these lines to just one logical expression

```suggestion
          {isSelectCategory && isSearchInputEmpty && (...)}
```

Comment (sjyothi54) [reply src/pages/index.jsx]: @josephgregoryii Updated the changes

File: src/pages/index.jsx [hunk 2]
```diff
@@ -427,7 +432,7 @@ const QuickstartsPage = ({ data, location }) => {
             </Overlay>
           </div>
 
-          {isSearchInputEmpty && (
+          {isSelectCategory && isSearchInputEmpty && (
```
Comment (aswanson-nr) [inline src/pages/index.jsx:435]: This approach totally works, it just adds some complexity with a new hook. Another option would be to key off of the `category` hook, which is already being/unset by the existing code. Then you can completely remove the need for a new `isSelectCategory` hook.
```suggestion
          {category === '' && isSearchInputEmpty && (
```

**Review by aswanson-nr — APPROVED** [review]
Hey @sjyothi54! I noticed a more simple way of achieving this and left a comment, but that change is not required. Feel free to merge!

**Pull request merged**